pkg/costmodel/cluster.go

```diff
@@ -146,7 +145,7 @@ func ComputeClusterCosts(client prometheus.Client, provider cloud.Provider, wind
 	hourlyToCumulative := float64(minsPerResolution) * (1.0 / 60.0)
 
 	const fmtQueryDataCount = `
-		count_over_time(sum(kube_node_status_capacity_cpu_cores) by (cluster_id)[%s:1m]%s)
+		count_over_time(sum(kube_node_status_capacity_cpu_cores) by (cluster_id)[%s:%dm]%s) * %d
```

Comment:
Previously, at startup this would return 1, 2, ..., 5, 6, 7, .... as the minutes pass. Now it will be less accurate, but will match the scaling of `hourlyToCumulative`, returning 5, 5, ..., 5, 10, 10, ... as the first minutes pass.

That query is used to create `dataMinsPerCluster`, which feeds the function `NewClusterCostsFromCumulative` to compute monthly costs.